### DISCUSSION TITLE
Found a new warning: "You are about to paste text that is longer than 5 KiB. Do you wish to continue?"

### DISCUSSION THREAD

**Discussion by 3SCadmin** [microsoft/terminal#15850] [Q&A] [answered]
There are plenty of resources informing us how to disable the "multiline paste" warning.
But this is different.  I get the Warning: "You are about to paste text that is longer than 5 KiB. Do you wish to continue?" when I paste a very long string (no line breaks).

Does anyone know the settings parameter to disable this warning?

**Comment by 3SCadmin** [comment, marked as answer]
Ok, I figured it out by finding the warning text in the source code, and applying tiny bit of logic
`"largePasteWarning" : false,`

**Reply by Benjamin-Loison** [reply]
[Here](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/interaction#warn-when-the-text-to-paste-is-very-large) is the associated documentation.
